### ISSUE TITLE
Hardhat project was not able to initialize correctly

### ISSUE THREAD

**Issue by Yurii921** [NomicFoundation/solidity-analyzer#19]
When initializing a Hardhat project with `npm install` using WSL 2, then the VS Code extension `NomicFoundation.hardhat-solidity` says that can not find module `@nomicfoundation/solidity-analyzer-win32-x64-msvc`.

Probably `@nomicfoundation/solidity-analyzer-win32-x64-msvc` is not installed because it thinks it is running on Ubuntu because of WSL.

`NomicFoundation.hardhat-solidity` version is 0.8.2.

`@nomicfoundation/hardhat-toolbox` version is 5.0.0.
`hardhat` version is 2.22.3.

The full error:
```
Hardhat project '...' was not able to initialize correctly:
 Cannot find module '@nomicfoundation/solidity-analyzer-win32-x64-msvc'
Require stack:
- ...\node_modules\@nomicfoundation\solidity-analyzer\index.js
- ...\node_modules\@nomicfoundation\ignition-core\dist\src\verify.js
- ...\node_modules\@nomicfoundation\ignition-core\dist\src\index.js
- ...\node_modules\@nomicfoundation\hardhat-ignition\dist\src\index.js
- ...\node_modules\@nomicfoundation\hardhat-ignition-ethers\dist\src\index.js
- ...\node_modules\@nomicfoundation\hardhat-toolbox\index.js
- ...\hardhat.config.ts
- ...\node_modules\hardhat\internal\core\config\config-loading.js
- ...\node_modules\hardhat\register.js
- ...\.vscode\extensions\nomicfoundation.hardhat-solidity-0.8.2\server\out\worker\WorkerProcess.js
```

**Comment by Yurii921** [comment]
Why completed? There is no fix or message.

**Comment by alcuadrado** [comment]
Sorry for closing it without a message. We now detect this error and explain how to solve it. On top of that, we found a definite solution.

**Comment by alcuadrado** [comment]
Fixed in 0.1.2

**Comment by Yurii277** [comment]
Thank you.
